tools/bin/commands/build

```diff
@@ -281,6 +282,7 @@ MODULE_PATHS=(
   "meta:app/meta/"
   "server:app/server/"
   "s2i:app/s2i/"
+  "teiid-syndesis:app/teiid-syndesis/"
```

Comment:
call as "dv", teiid-syndesis is only repo name, when "dv" should be used else where to represent anything else.